recipes/ctapipe-io-nectarcam/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "ctapipe-io-nectarcam" %}
+{% set version = "0.1.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/ctapipe_io_nectarcam-{{ version }}.tar.gz
+  sha256: 3578fe5cc354e8a729430964ba7fd6c936eaf112733082e3d8f3d499a1589bd3
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+  skip: true  # [win]
+
+requirements:
+  host:
+    - python =3.8
+    - setuptools >=64.0.3
+    - wheel
+    - setuptools-scm >=3.4
+    - pip
+  run:
+    - python =3.8
+    - astropy =4.2
+    - ctapipe =0.12
+    - numpy =1.22
+    - protozfits =2.0
```

Comment:
Does the package really only work with these exact versions? Being so specific tends to lead to difficult-to-resolve environments if this package is installed alongside anything else in an environment.

Reply:
Yes, we are aware of that. Since this package integrates itself within an environment for a future gamma-ray observatory, all our softwares are in heavy development phase (v0.x.x phase) with frequent API breaking changes. This package heavily relies on `ctapipe`, for instance, and will be tightly coupled to it's different versions.

Reply:
Ok, but why such a tight pin on numpy, astropy, and python?

Reply:
A new recipe with relaxed constraint on `astropy`, `numpy` and `python` is now proposed.

Reply:
👍